src/models/fill_in_the_blanks.rs

```diff
@@ -45,6 +45,10 @@ impl FillInTheBlanks {
     fn next(&mut self) {
         self.blank_index = self.blank_index % self.answers.len();
     }
+
+    pub fn validate_answer(&mut self) -> Option<bool> {
+        None
```

Comment:
> Log::Warn "Implement me!"

not sure how to do that in Rust... we may not have a log file at the moment but maybe an issue I can create down the line to keep logs!

Reply:
As per our conversation, this will be fixed by creating a trait where the logic for that will go